integration-tests/features/component_analysis_v2.feature

```diff
@@ -0,0 +1,246 @@
+     | maven      | io.vertx:vertx-core | 3.5.2.CR2 | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-32470     | 5.3    | 
+     | maven      | io.vertx:vertx-core | 3.5.2.CR1 | 4.0.0-milestone4    | SNYK-JAVA-IOVERTX-32470     | 5.3    |
+     | pypi       | flask               | 0.12      | 1.1.2               | SNYK-PYTHON-FLASK-42185     | 7.5    |
+     | pypi       | flask               | 0.12.1    | 1.1.2               | SNYK-PYTHON-FLASK-42185     | 7.5    |
+     | pypi       | flask               | 0.12.2    | 1.1.2               | SNYK-PYTHON-FLASK-42185     | 7.5    |
+     | pypi       | numpy               | 1.16.0    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.4    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.3    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.2    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | numpy               | 1.15.1    | 1.18.4              | SNYK-PYTHON-NUMPY-73513     | 9.8    |
+     | pypi       | requests            | 2.19.1    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
+     | pypi       | requests            | 2.19.0    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
+     | pypi       | requests            | 2.18.4    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
+     | pypi       | requests            | 2.18.3    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
+     | pypi       | requests            | 2.18.2    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
+     | pypi       | requests            | 2.17.3    | 2.23.0              | SNYK-PYTHON-REQUESTS-72435  | 9.8    |
+
+ 
+  
+  Scenario Outline: Check the component analysis V2 REST API endpoint for components without recommendations
+      Given System is running
+      Given Three scale preview service is running
+    
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+      When I start v2 component analyses <ecosystem>/<package>/<version> with user_key
+      Then I should get 200 status code
+        And I should receive a valid JSON response
+      Then I should find no recommendation
+
+
+      Examples: EPV
+     | ecosystem  | package  | version |
+     | npm | sequence | 3.0.0 |
+     | npm | aargh | 1.1.0 |
+     | npm | arrays | 0.1.1 |
+     | npm | jquery | 3.5.0 |
+     | npm | mocha | 6.1.4 |
+     | npm | underscore | 1.9.1 |
+     | npm | babel-core | 6.26.3 |
+     | npm | babel-core | 6.26.2 |
+     | npm | babel-core | 7.0.0-bridge.0 |
+     | npm | babel-core | 7.0.0-beta.3 |
+     | npm | babel-core | 7.0.0-beta.2 |
+     | npm | babel-core | 7.0.0-beta.1 |
```

Comment:
similar to what i mentioned above. Rather than repeating the test for the same package, we can use some other package. It doesnt make sense to me. Looks like we will simply be wasting e2e time for doing the same test

Reply:
as i said earlier followed the V1 approach 
and also about e2e run time , without component analysis v2 tests it is 20 mins 
with v2 it is 22 mins 
new tests run faster because they dont need any sleep time or wait time

Reply:
same as above

Reply:
I dont think we need to check all versions of babel-core

Reply:
okay will reduce them